cli: example fails on 58a68a

On the current `HEAD` of the GWpy repository, the CLI example `cli-spg-01.png` fails to build as follows:

```
$ make ../build/sphinx/cli_examples/cli-spg-01.png
cli-spg-01
Traceback (most recent call last):
  File "../bin/gwpy-plot", line 118, in <module>
    result_code = prod.makePlot(args)
  File "/home/duncan.macleod/.local/lib/python2.6/site-packages/gwpy-0.1b1.dev17-py2.6.egg/gwpy/cli/cliproduct.py", line 797, in makePlot
    self.gen_plot(args)
  File "/home/duncan.macleod/.local/lib/python2.6/site-packages/gwpy-0.1b1.dev17-py2.6.egg/gwpy/cli/spectrogram.py", line 94, in gen_plot
    snr_nfft = self.dur / (secpfft * stride)
NameError: global name 'stride' is not defined
make: *** [../build/sphinx/cli_examples/cli-spg-01.png] Error 1
```

@areeda, can you take a look?

--- a/gwpy/cli/coherence.py
+++ b/gwpy/cli/coherence.py
@@ -121,26 +121,30 @@ class Coherence(CliProduct):
                         if next_ts != ref_idx:
                             if numpy.min(self.timeseries[next_ts]) == \
                                     numpy.max(self.timeseries[next_ts]):
-                                print 'Channel %s at %d has min=max, coherence ' \
-                                      'with this channel will not be calculated' \
-                                      % self.timeseries[next_ts].channel.name, \
+                                print 'Channel %s at %d has min=max, ' \
+                                      'coherence with this channel will not ' \
+                                      'be calculated' \
+                                      % self.timeseries[next_ts].channel.name,\
                                     self.timeseries[next_ts].times.epoch.gps
                             else:
                                 maxfs = max(maxfs,
-                                            self.timeseries[next_ts].sample_rate)
+                                            self.timeseries[next_ts].
+                                            sample_rate)
                                 # calculate and plot the first pair,
-                                # note the first channel is the reference channel
+                                # note the first channel is the reference chan
                                 snd_ts = self.timeseries[next_ts]
                                 coh = self.timeseries[ref_idx].\
                                     coherence(snd_ts, fftlength=fftlen,
                                               overlap=ovlap_frac*fftlen)
 
-                                legend_text = self.timeseries[next_ts].channel.name
+                                legend_text = self.timeseries[next_ts].\
+                                    channel.name
                                 if len(self.start_list) > 1:
-                                    legend_text += ", %s" % snd_ts.times.epoch.gps
+                                    legend_text += ", %s" % \
+                                                   snd_ts.times.epoch.gps
                                 coh.name = legend_text
 
-                                # coh2 = 1 / (1-coh) : how to implement alt scaler
+                                # coh2 = 1 / (1-coh) : how to do alt scaler
 
                                 if not cohs:
                                     self.plot = coh.plot()

--- a/gwpy/cli/spectrogram.py
+++ b/gwpy/cli/spectrogram.py
@@ -83,21 +83,26 @@ class Spectrogram(CliProduct):
         ovlp_sec = secpfft*ovlp_frac
         nfft = self.dur/(secpfft - ovlp_sec)
         fft_per_stride = int(nfft/(self.width * 0.8))
-        stride_sec = fft_per_stride * (secpfft - ovlp_sec) + secpfft -1
+        stride_sec = fft_per_stride * (secpfft - ovlp_sec) + secpfft - 1
         stride_sec = max(2*secpfft, stride_sec)
         fs = self.timeseries[0].sample_rate.value
-        self.log(3, ('Spectrogram calc, stride: %.2f, fftlength: %.2f, '
-                'overlap: %.2f' % (stride_sec, secpfft, ovlp_sec)))
 
         # based on the number of FFT calculation choose between
         # high time resolution and high SNR
         snr_nfft = self.dur / stride_sec
         if (snr_nfft > 512):
-            specgram = self.timeseries[0].spectrogram(stride_sec, fftlength=secpfft,
+            specgram = self.timeseries[0].spectrogram(stride_sec,
+                                                      fftlength=secpfft,
                                                       overlap=ovlp_sec)
+            self.log(3, ('Spectrogram calc, stride: %.2fs, fftlength: %.2f, '
+                         'overlap: %.2f, #fft: %d' %
+                         (stride_sec, secpfft, ovlp_sec, snr_nfft)))
         else:
             specgram = self.timeseries[0].spectrogram2(fftlength=secpfft,
                                                        overlap=ovlp_sec)
+            self.log(3, ('HR-Spectrogram calc, stride: %.2fs, fftlength: %.2f,'
+                         ' overlap: %.2f, #fft: %d' %
+                         (stride_sec, secpfft, ovlp_sec, snr_nfft)))
         specgram = specgram ** (1/2.)   # ASD
 
         norm = False

--- a/gwpy/cli/spectrogram.py
+++ b/gwpy/cli/spectrogram.py
@@ -91,7 +91,7 @@ class Spectrogram(CliProduct):
 
         # based on the number of FFT calculation choose between
         # high time resolution and high SNR
-        snr_nfft = self.dur / (secpfft * stride)
+        snr_nfft = self.dur / stride_sec
         if (snr_nfft > 512):
             specgram = self.timeseries[0].spectrogram(stride_sec, fftlength=secpfft,
                                                       overlap=ovlp_sec)

